### ISSUE TITLE
Incompatible with object color selector.

### ISSUE THREAD

**Issue by thomascswalker** [curtwartenberg/LlamaTools-3dsMax-Plugins#2]
![image](https://user-images.githubusercontent.com/13595958/102147518-dfdfaa00-3e38-11eb-9739-60046f480c39.png)

Not sure how you resolve this, but it's something I came across.

**Comment by curtwartenberg** [comment]
Correct, this messagebox was my solution to this because 3ds max would crash on close.
